src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stringsArr = sourceString.split(';');
+  const res = {};
+
+  stringsArr.forEach((str) => {
+    const clSrt = str.trim();
```

Comment:
Consider renaming `clSrt` to `cleanStr` or `cleanedStr` for better readability and understanding of its purpose.